app/src/androidTest/java/org/simple/clinic/storage/migrations/Migration71AndroidTest.kt

```diff
@@ -0,0 +1,37 @@
+package org.simple.clinic.storage.migrations
+
+import org.junit.Test
+import org.simple.clinic.assertColumns
+
+class Migration71AndroidTest : BaseDatabaseMigrationTest(70, 71) {
+
+  @Test
+  fun registered_facility_id_and_assigned_facility_id_should_be_added_to_patient_table() {
+    before.assertColumns("LoggedInUser", setOf(
```

Comment:
Can we also create a default entry and run the migration to ensure that the values are retained after migration correctly?